launch/vision.launch

```diff
@@ -2,12 +2,52 @@
 
 <launch>
 
+    <arg name="doImShow" default="false"/>
+    <arg name="leftImage" default="camera/left/undistorted"/>
+    <arg name="rightImage" default="camera/right/undistorted"/>
+    
+    <arg name="simulated" default="false"/>
+
+    <rosparam param="/visualize_thrusters" subst_value="True" if="$(arg simulated)">false</rosparam>
```

Comment:
Add a comment here for why it is necessary to change the value of this parameter.